From fac8e6b2f79e8299eec7d19eb8076aac47276042 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Sat, 3 Dec 2022 13:02:08 +0200
Subject: [PATCH] GLODROID: Don't add -ljpeg argument when building libyuv as
 subproject

We don't have sysroot with libdir when building within AOSP.
Instead all dependencies are linked directly as path/lib.so link argument.

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 src/android/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/android/meson.build b/src/android/meson.build
index 1bba54de..b1b5caa1 100644
--- a/src/android/meson.build
+++ b/src/android/meson.build
@@ -29,7 +29,6 @@ if not libyuv_dep.found()
          '-Wno-sign-compare',
          '-Wno-unused-variable',
          '-Wno-unused-parameter')
-    libyuv_vars.append_link_args('-ljpeg')
     libyuv = cmake.subproject('libyuv', options : libyuv_vars)
     libyuv_dep = libyuv.dependency('yuv')
 endif
-- 
2.37.2

